Purchasing.Mvc/Controllers/AdminController.cs

```diff
@@ -588,12 +589,14 @@ public virtual bool NeedToCheckWorkgroupPermissions(string key)
                 {
                     var sgMessage = new MailMessage(
                         new MailAddress("opp-noreply@ucdavis.edu", "OPP No Reply"),
-                        new MailAddress("opp-tech@ucdavis.edu"))
+                        new MailAddress("apprequests@caes.ucdavis.edu"))
                     {
                         Subject = "Check Workgroup Permissions",
                         Body = "Run the Check"
                     };
 
+                    Log.Error("Run the Workgroup Permissions Check");
```

Comment:
@jSylvestre I think it should be a warn since it doesn't actually fit the exception/error concept